apps/cli/tf/outputs.tf

```
@@ -0,0 +1,4 @@
+output "GITHUB_IAM_ROLE_ARN" {
+value = module.iam_iam-assumable-role-with-oidc.iam_role_arn
+description = "Connect to the S3 bucket from github with this arn"
```

Proper indentation.

```suggestion
  value = module.iam_iam-assumable-role-with-oidc.iam_role_arn
  description = "Connect to the S3 bucket from github with this arn"
```

Let's consider using a linter/formatter, if tf doesn't have one, we could use the current prettier configuration in the repository if it supports tf.

---

We have these options : 

1 - **Terraform fmt**: 
`Terraform includes its built-in formatter called terraform fmt. It automatically updates and formats our Terraform code according to the Terraform style conventions. we can run terraform fmt command in our project directory to format the code.`

2 - **TFLint**: 
`TFLint is a Terraform linter that analyzes our code and provides suggestions, warnings, and errors based on a set of predefined rules. It helps identify potential issues or deviations from best practices. we can integrate TFLint into our CI/CD pipeline or run it locally. More information about TFLint can be found at: https://github.com/terraform-linters/tflint`

3 - **Checkov**: 
`Checkov is an open-source static analysis tool for infrastructure-as-code (IaC) files. It supports multiple IaC frameworks, including Terraform, and scans our code to identify security and compliance issues. Checkov can be integrated into our development workflow or run as a standalone tool. More information about Checkov can be found at: https://www.checkov.io/`

4 - **pre-commit**: 
`pre-commit is a framework that allows us to automatically run various linters, including Terraform linters, before each commit. It helps enforce code quality standards and prevents committing code that doesn't pass the linter checks. we can configure pre-commit to run Terraform linters like terraform fmt, TFLint, or Checkov. More information about pre-commit can be found at: https://pre-commit.com/`